rules-tests/CodeQuality/Rector/ClassMethod/ExplicitReturnNullRector/Fixture/do_while_maybe_returned4.php.inc

```diff
@@ -17,28 +17,3 @@ final class DoWhileMaybeReturned4
 			echo 'test';
     }
 }
-
-?>
------
-<?php
-
-namespace Rector\Tests\CodeQuality\Rector\ClassMethod\ExplicitReturnNullRector\Fixture;
-
-final class DoWhileMaybeReturned4
-{
-    public function run(int $i)
-    {
-		do {
-			if (rand(0, 1)) {
-				goto execute;
-			}
-
-			return 2;
-		} while (++$i < 1);
-		execute:
-			echo 'test';
-            return null;
-    }
-}
-
-?>
```

Comment:
hopefully its fine to skip this case, as I think it is not very useful

Reply:
I agree it is technically allowed, but I think its not valuable to complicate the rector for goto cases

Reply:
I created PR with utilize `TerminatedNodeAnalyzer` at:

- https://github.com/rectorphp/rector-src/pull/6990

Reply:
After some try, it indeed to complicated to tweak, I can merge this for now :)